storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.4.rst

```diff
@@ -0,0 +1,23 @@
+============================
+ |Percona XtraBackup| 2.3.4
+============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.4 on March 17th 2015. Downloads are available from our download site `here <http://www.percona.com/downloads/XtraBackup/xtrabackup-2.3.4/>`_. Downloads are available from our `download site <http://www.percona.com/downloads/XtraBackup/LATEST/>`_ and from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3 series.
+
+Bugs fixed
+----------
+
+ |Percona XtraBackup| didn't respect variables stored in :file:`backup-my.cnf` unless they were specified with :option:`xtrabackup --defaults-file` option. Bug fixed :bug:`1527081`.
```

Comment:
unless they were specified -> unless it was specified. One was needed to specify --defaults-file=backup-my.cnf for options to be respected.